pages/report-repair/[route].js

```diff
@@ -181,6 +181,7 @@ function ReportRepair() {
       if (response.ok) {
         setShowBack(false);
         router.push('repair-cancelled-confirmation');
+        setConfirmation(values.repairAppointmentDetails.contactDetails.value);
```

Comment:
This is not required as the cancellation page (no longer) shows the contact details.

(I feel this and the above [comment](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/107/files#diff-54de87070147bd6483cb396dde9b615c82d0684d7ef8dd228e327ea5b0d36ddcR107) may be accidental and caused by merge conflicts)